Automattic/jetpack
# Product management controls: Remove dropdown indication

We have removed the dropdown icon from the product management controls subscription selector.




Fixes #23880

#### Changes proposed in this Pull Request:

* We've removed the dropdown indication.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Create or edit a new post.
* Add a premium content block.
* The controls should not display the dropdown icon in the subscription selector.

#### Test evidence

https://user-images.githubusercontent.com/1989914/164178022-1847e7f4-8bdd-4c9c-af56-90793a35d10a.mp4
